frontend/src/components/Pages/MissionPage/MapPosition/MissionMapView.tsx

```diff
@@ -67,7 +67,7 @@ export function MissionMapView({ mission }: MissionProps) {
     }
 
     const findCurrentTaskOrder = useCallback(() => {
-        mission.tasks.forEach(function (task) {
+        mission.tasks.forEach((task) => {
             if (task.status === TaskStatus.InProgress || task.status === TaskStatus.Paused) {
                 setCurrentTaskOrder(task.taskOrder)
```

Comment:
Unrelated to your PR, but this isn't necessarily safe. Sorry you can ignore this comment, this is moreso a reminder to myself :)